apps/condo/domains/user/components/auth/InputPhoneForm.tsx

```diff
@@ -94,57 +140,41 @@ export const InputPhoneForm: React.FC<IInputPhoneFormProps> = ({ onFinish })=> {
             labelAlign='left'
             requiredMark={false}
         >
-            <Row gutter={[0, 60]}>
+            <Row gutter={GUTTER_0_60}>
                 <Col span={24}>
-                    <Row gutter={[0, 40]}>
+                    <Row gutter={GUTTER_0_40}>
```

Comment:
It's better to leave such consts here. It's not in increase the readability

Reply:
removed all refactoring